apiary.apib

```diff
@@ -436,7 +439,7 @@ Available sorting and filtering fields:
            
 + Response 200 
 
-    Returning list of found investments on secondary marketplace
```

Comment:
How this relates to insurance?

I'm not sure why `users/me/investments` section is grouped under secondary marketplace at all, but that should take another review. 🤔

Reply:
I think that this group of resources has been published when secondary marketplace was rolled out for a scenario of selling your investments on SMP. Maybe we could rearrange it a bit to fit multiple scenarios later.